app/containers/main-container.jsx

```diff
@@ -50,7 +55,9 @@ const mapStateToProps = state => ({
 
 const mapDispatchToProps = dispatch => ({
   stepAddGoal: () => { dispatch(action_stepAddGoal()) },
-  onInputGoal: (text) => { dispatch(action_inputGoal(text)) },
+  onInputGoal: (text) => { dispatch(actionsAddNewGoal.inputGoal(text)) },
```

Comment:
@des-des can you explain a bit more please? not sure what i'm supposed to be doing...